onnxruntime/core/providers/migraphx/migraphx_execution_provider.cc

```diff
@@ -992,21 +1003,22 @@ GetUnsupportedNodeIndices(const GraphViewer& graph_viewer,
                           /*out*/ std::unordered_set<std::string>& mgx_required_initializers,
                           const logging::Logger& logger) {
   static std::set<std::string> mgx_supported_ops = {"Abs", "Acos", "Acosh", "Add", "And", 
```

Comment:
>mgx_supported_ops [](http://example.com/codeflow?start=31&length=17)

hi, do you have the pointers where I can find the current supported ops? thx

Reply:
Do you mean you want to check the ops that are supported by migraphx?

Reply:
Yes, is there any online docs or source code indicate current supported ops?

Reply:
If you can build the migraphx source, then in the build folder, you can get all the supported ops with the following command line:

```
 scxiao@scdesktop:~/Workplace/projects/AMDMIGraphX/build$ ./bin/migraphx-driver onnx -l
```

Building the migraphx source can follow (two commands: 1) install ```rbuild```, then build the source code): 

```
https://github.com/ROCmSoftwarePlatform/AMDMIGraphX#use-the-rocm-build-tool-rbuild
```

it will print out all the ONNX operators that are supported. 
I will have the team to add a webpage to indicate the OPs that we can support, then you can just refer that directly.

Reply:
Awesome, thanks for sharing !!